src/backend/InvenTree/build/api.py

```diff
@@ -529,19 +535,6 @@ class BuildLineMixin:
 
     def get_serializer(self, *args, **kwargs):
         """Return the serializer instance for this endpoint."""
-        kwargs['context'] = self.get_serializer_context()
-
-        try:
-            params = self.request.query_params
-
-            kwargs['bom_item_detail'] = str2bool(params.get('bom_item_detail', True))
-            kwargs['assembly_detail'] = str2bool(params.get('assembly_detail', True))
-            kwargs['part_detail'] = str2bool(params.get('part_detail', True))
-            kwargs['build_detail'] = str2bool(params.get('build_detail', False))
-            kwargs['allocations'] = str2bool(params.get('allocations', True))
-        except AttributeError:
-            pass
-
         return super().get_serializer(*args, **kwargs)
```

Comment:
We could remove this get_serializer function now as it is empty

Reply:
Let's remove it for now; once your merge-train is done, I can take a look at that mechanism in general